Source/AliveLibAO/Abe.cpp

```diff
@@ -2738,11 +2740,393 @@ void Abe::VOn_Tlv_Collision_421130(Path_TLV *pTlv)
+        }
+        return;
+    }
+    if (sInputObject_5009E8.isPressed(sInputKey_FartRoll_4C65B0))
+    {
+        field_FC_current_motion = eAbeStates::State_21_StandToCrouch_427F40;
+    }
+
+    bool handleDoActionOrThrow = false;
+    if (sInputObject_5009E8.isPressed(sInputKey_Up_4C6598))
+    {
+        if (field_F8_pLiftPoint)
+        {
+            if (field_F8_pLiftPoint->field_4_typeId == Types::eLiftPoint_51)
+            {
+                const FP halfGrid = ScaleToGridSize_41FA30(field_BC_sprite_scale) / FP_FromInteger(2);
+                const FP liftPlatformXMidPoint = FP_FromInteger((field_F4_pLine->field_0_rect.x + field_F4_pLine->field_0_rect.w) / 2);
+                const FP xPosToMidLiftPlatformDistance = FP_Abs(field_A8_xpos - liftPlatformXMidPoint);
+                if (xPosToMidLiftPlatformDistance < halfGrid)
+                {
+                    //AO exclusive - Abe only uses lift facing one side
+                    if (field_10_anim.field_4_flags.Get(AnimFlags::eBit5_FlipX))
+                    {
+                        field_FC_current_motion = eAbeStates::State_2_StandingTurn_426040;
+                        field_FE_next_state = eAbeStates::State_133_LiftGrabBegin_42EF20;
+                    }
+                    else
+                    {
+                        field_FC_current_motion = eAbeStates::State_133_LiftGrabBegin_42EF20;
+                    }
+                    return;
+                }
+            }
+        }
+
+        field_FC_current_motion = TryMountElum_42E600();
+        if (field_FC_current_motion != eAbeStates::State_0_Idle_423520)
+        {
+            return;
+        }
+        pTlv = gMap_507BA8.TLV_Get_At_446060(
+            nullptr,
+            field_A8_xpos,
+            field_AC_ypos,
+            field_A8_xpos,
+            field_AC_ypos
+        );
+
+        while (pTlv)
+        {
+            switch (pTlv->field_4_type)
+            {
+                case TlvTypes::Door_6:
+                {
+                    if (NearDoorIsOpen() && !field_10A_flags.Get(Flags_10A::e10A_Bit5_Electrocuted))
+                    {
+                        field_F0_pTlv = pTlv;
+                        field_110_state = 0;
+                        field_FC_current_motion = eAbeStates::State_156_DoorEnter_42D370;
+
+                    }
+                    else
+                    {
+                        if (sInputObject_5009E8.isHeld(sInputKey_Up_4C6598))
+                        {
+                            field_FC_current_motion = eAbeStates::State_36_DunnoBegin_423260;
+                        }
+                    }
+                    return;
+                }
+                case TlvTypes::LocalWell_11:
+                {
+                    auto well = static_cast<Path_Well_Local*>(pTlv);
+                    if ((well->field_18_scale || field_BC_sprite_scale != FP_FromInteger(1)) &&
```

Comment:
same as the other comment, needs inverting to match AE I think

Reply:
It doesn't, same as in AE.